ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/user-labels-tags/provision/cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision-chain.yaml

```diff
@@ -52,10 +51,7 @@ chain:
       openshift-qe X X 
       openshift-qe hello.Su_Zhe No. 6
       openshift-qe Su.Shi-Ding_Feng_Bo Mo Ting Chuan Lin Da Ye Sheng
-      openshift-qe This.is.test-message1234567890.1234567890.1234567890.1234567890 hello_a-b.c@d%e=f+g:h,i*j#k&l(m){n}[o](p){q}[r]@@@@@@@@@@@@@@@z
-      openshift-qe TEST-KEY02 TEST_VALUE-02
```

Comment:
What's wrong with line 56? It doesn't have comma character.

Reply:
Maybe my mistake. Add it back.